run.go

```diff
@@ -63,10 +63,12 @@ func run(c *cli.Context) error {
 		}
 	}()
 
+	group, ctx := errgroup.WithContext(ctx)
```

Comment:
As this is a bugfix, maybe it's cleaner to make it the first commit. Before adding new functionality.

Reply:
moved to first commit!